### DISCUSSION TITLE
Rectangles in prompt instead of icons, both Terminal icons and font are installed

### DISCUSSION THREAD

**Discussion by ravjotsandhu** [JanDeDobbeleer/oh-my-posh#3376] [Troubleshoot]
![image](https://user-images.githubusercontent.com/58950644/213467173-9e8b052b-f8db-40b4-9405-051fc5326137.png)

power shell profile
![image](https://user-images.githubusercontent.com/58950644/213467060-a9bc0a09-ff89-4ee3-b4f2-ea0356b159d3.png)

I read FAQ but that isn't helping either https://ohmyposh.dev/docs/faq#there-are-rectangles-instead-of-icons-in-my-prompt

![image](https://user-images.githubusercontent.com/58950644/213466662-19e9253d-ecd7-4027-9177-3026b7f05acd.png)

**Comment by JanDeDobbeleer** [comment]
Did you install the font you want to use? The names aren't always as straightforward and you might want to look in the fonts folder to see the name you have to use. But once configured, it alls works automagically.

**Reply by ravjotsandhu** [reply]
Yes! Look at topmost screenshot

**Comment by ravjotsandhu** [comment]
rechecked name but still doesnot work
**Terminal setttings**👇
![image](https://user-images.githubusercontent.com/58950644/213488705-b5b15b91-7371-4ac6-993a-6d993e4ee9d4.png)

**font name**👇
![image](https://user-images.githubusercontent.com/58950644/213488850-f411bfef-4149-49db-afb9-7df3d85cf3af.png)

**error**👇
![image](https://user-images.githubusercontent.com/58950644/213489013-c9eb7433-7ce4-4bad-9925-0ff65f9366d2.png)

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu that's a terminal question and not related to oh-my-posh to be honest. That configuration isn't something we influence.

**Reply by ravjotsandhu** [reply]
@JanDeDobbeleer  Anything else I might be missing? Or is it just because of font problem that I am getting rectangles instead of icons in powershell

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu correct. Solving the font configuration issue solves the rectangles.

**Reply by ravjotsandhu** [reply]
https://github.com/JanDeDobbeleer/oh-my-posh/issues/145#issuecomment-761057261 
I found this, I had already copied them but deleted and did again just as this gif suggests but still I get font not found

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu did you restart Windows Terminal and even Windows? Could be it's not loaded into the terminal.

**Reply by ravjotsandhu** [reply]
Yup, I did

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu can you raise an issue at https://github.com/microsoft/terminal as this is beyond our scope.

**Comment by ravjotsandhu** [comment]
Hey @JanDeDobbeleer I got it working with MesloLGS font but with Agave its not working

I went to powershell separatly and checked them there
![image](https://user-images.githubusercontent.com/58950644/214483049-062c053f-f70e-446a-ad6d-626219735f81.png)
and then corrected in my terminal settings too
![image](https://user-images.githubusercontent.com/58950644/214483101-7e5aa917-321e-49ca-aee3-ee27a83bd9f8.png)

![image](https://user-images.githubusercontent.com/58950644/214483681-1033b362-dabb-40b3-9ba0-d7fb32a31744.png)


_**BUT** for Agave it still shows rectangles_ 👇
![image](https://user-images.githubusercontent.com/58950644/214484172-4fb551c6-a6cc-4fc3-bcb6-0e25a37cb7bb.png)

![image](https://user-images.githubusercontent.com/58950644/214484108-fa91b5da-bc57-4996-bab0-9cabb6977a2e.png)

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu no idea why that happens, that's a question for Windows Terminal. They are responsible for this.

**Comment by ravjotsandhu** [comment]
And neither works in vscode 
![image](https://user-images.githubusercontent.com/58950644/214485416-5e75c1c5-d8d1-4083-9e45-2a22d2a7ed33.png)

**Reply by ravjotsandhu** [reply]
@JanDeDobbeleer what about this?

**Reply by JanDeDobbeleer** [reply]
@ravjotsandhu you PATH isn't loaded in vscode for some reason. That's a question for vscode.
